atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownPutTest.java

```diff
@@ -31,73 +30,65 @@
-        OneNodeDownTestSuite.kvs.putUnlessExists(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableMap.of(OneNodeDownTestSuite.CELL_4_1, OneNodeDownTestSuite.DEFAULT_CONTENTS));
-        OneNodeDownTestSuite.verifyValue(OneNodeDownTestSuite.CELL_4_1,
-                Value.create(OneNodeDownTestSuite.DEFAULT_CONTENTS, AtlasDbConstants.TRANSACTION_TS));
+        getTestKvs().putUnlessExists(TEST_TABLE, ImmutableMap.of(EMPTY_CELL, CONTENTS));
+        assertLatestValueInCell(EMPTY_CELL, Value.create(CONTENTS, AtlasDbConstants.TRANSACTION_TS));
     }
 
     @Test
     public void putUnlessExistsThrowsOnExists() {
-        OneNodeDownTestSuite.kvs.putUnlessExists(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableMap.of(ATOMIC_CELL, OneNodeDownTestSuite.DEFAULT_CONTENTS));
-
-        assertThatThrownBy(() -> OneNodeDownTestSuite.kvs.putUnlessExists(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableMap.of(ATOMIC_CELL, NEW_CONTENTS)))
+        byte[] newContents = PtBytes.toBytes("new_value");
+        assertThatThrownBy(() -> getTestKvs().putUnlessExists(TEST_TABLE, ImmutableMap.of(NONEMPTY_CELL, newContents)))
                 .isInstanceOf(KeyAlreadyExistsException.class);
 
-        Map<Cell, Value> result = OneNodeDownTestSuite.kvs.get(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableMap.of(ATOMIC_CELL, AtlasDbConstants.TRANSACTION_TS));
-        assertThat(Value.create(NEW_CONTENTS, AtlasDbConstants.TRANSACTION_TS))
-                .isNotEqualTo(result.get(ATOMIC_CELL));
+        Map<Cell, Value> result = getTestKvs()
+                .get(TEST_TABLE, ImmutableMap.of(NONEMPTY_CELL, AtlasDbConstants.TRANSACTION_TS));
+        assertThat(result.get(NONEMPTY_CELL)).isNotEqualTo(Value.create(newContents, AtlasDbConstants.TRANSACTION_TS));
     }
 
     @Test
     public void canAddGarbageCollectionSentinelValues() {
-        OneNodeDownTestSuite.kvs.addGarbageCollectionSentinelValues(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableSet.of(OneNodeDownTestSuite.CELL_3_1));
-        Map<Cell, Long> latestTimestamp = OneNodeDownTestSuite.kvs.getLatestTimestamps(OneNodeDownTestSuite.TEST_TABLE,
-                ImmutableMap.of(OneNodeDownTestSuite.CELL_3_1, Long.MAX_VALUE));
-        assertEquals(Value.INVALID_VALUE_TIMESTAMP,
-                latestTimestamp.get(OneNodeDownTestSuite.CELL_3_1).longValue());
+        getTestKvs().addGarbageCollectionSentinelValues(TEST_TABLE, ImmutableSet.of(CELL_2_2));
+        Map<Cell, Long> latestTimestamp = getTestKvs()
+                .getLatestTimestamps(TEST_TABLE, ImmutableMap.of(CELL_2_2, Long.MAX_VALUE));
+        assertThat(latestTimestamp.get(CELL_2_2)).isEqualTo(Value.INVALID_VALUE_TIMESTAMP);
+    }
+
+    private void assertLatestValueInCell(Cell cell, Value value) {
+        Map<Cell, Value> result = getTestKvs().get(TEST_TABLE, ImmutableMap.of(cell, Long.MAX_VALUE));
+        assertThat(value).isEqualTo(result.get(cell));
     }
 }
```

Comment:
This test has always been nasty (and still seems a little so, though is much improved) in that test methods have side effects that hang around for the life of the test. Thanks for cleaning it up!

Reply:
Yes, it's kind of hard to avoid because to clean up properly we'd need to truncate the table and we don't have all the nodes up. Maybe drop and create the table? (j/k)